### PR TITLE
Compatiable with psr/container

### PR DESCRIPTION
**Pull request by FuriosoJack**
With compatible with interface container psr

### CODE REVIEW

File: bin/patch-type-declarations [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        }
+    }
+
+    if (function_exists('stream_wrapper_register') && stream_wrapper_register('phpvfscomposer', 'Composer\BinProxyWrapper')) {
+        include("phpvfscomposer://" . __DIR__ . '/..'.'/vendor/symfony/error-handler/Resources/bin/patch-type-declarations');
+        exit(0);
+    }
+}
+
+include __DIR__ . '/..'.'/vendor/symfony/error-handler/Resources/bin/patch-type-declarations';
```
Comment (ruudk) [inline bin/patch-type-declarations:107]: What's the purpose of this file? Looks like you committed this by accident.

Comment (FuriosoJack) [reply bin/patch-type-declarations:107]: Woow yes, I will create another commit to remove it-

Comment (FuriosoJack) [reply bin/patch-type-declarations:107]: > What's the purpose of this file? Looks like you committed this by accident.

but you know I think that this file is generated when I execute any of the commands that you have in the contribution document

Comment (ruudk) [reply bin/patch-type-declarations:107]: Then the file should probably be added to the .gitignore file.

Please rebase so that there is only 1 commit.

**Pull request closed without merging**